queries/java/highlights.scm

```diff
@@ -150,25 +168,30 @@
 "case"
 ] @conditional
 
-; 
+;
 
 [
 "for"
 "while"
 ] @repeat
```

Comment:
Is there a do...while loop in Java? So do would be also repeat.

Reply:
done